### ISSUE TITLE
You do not have permission to view OAuth Client with id

### ISSUE THREAD

**Issue by ghost** [Patreon/patreon-js#30]
Hi,

I'm following your code examples to set up authentication on my website. The redirect works fine and I'm receiving a token back, but the next call into the library throws an error:

```
{
  "code":null,
  "code_name":"OAuthClientViewForbidden",
  "detail":"You do not have permission to view OAuth Client with id <my_id>",
  "id":"<my_id>",
  "status":"403",
  "title":"You do not have permission to view this OAuth Client."
}
```

Is this library perhaps out of date with the server? Should I create a new client with version 1 instead of version 2?

Thank you!

**Comment by phildini** [comment]
Hi there! Does the `id` in `"detail"` match the `id` in `"id"`?

**Comment by ghost** [comment]
In "detail" I see my client id (matches the one in the portal as expected).
In "id" I see a GUID of some sort; I can paste it here if it doesn't leak any secrets.

**Comment by phildini** [comment]
Yeah, the `id` is a request ID that doesn't leak anything. If you can paste that here we can help dig in.

**Comment by ghost** [comment]
The request ID for one of the failed requests: `35dc31b8-a5b6-4bf4-99ea-4cfe48701cbf`

**Comment by uliantsev-a** [comment]
@IAmPicard maybe you already decision your ask, yet i think you wrong used library. 
This error message actualy with request of current_user when use token for not application's user.
For get information about connected user need use `identity` endpoint.

**Comment by rkumar1310** [comment]
I am facing the same issue here, the auth is working with one app but when I switch to another app I get the error, both app have same configuration. I am trying to call `/current_user` and it works for one app but not the other app. request id, `f026b239-2dc3-5ef8-a646-23e003981096`
